solutions/devicesimulation/armtemplate/template.json

```diff
@@ -219,10 +219,10 @@
                 "description": "The Azure Active Directory domain used by the Azure Subscription where the solution is deployed"
             }
         },
-        "subscriptionId": {
```

Comment:
I'm not sure I understand this, do we need to remove the subscription id from the list of parameters?

Reply:
oops, added it back